streams.yml

```diff
@@ -124,10 +124,10 @@ rhel8:
   mirror: true
 
 rhel9:
-  # the most recent release at present. since we yum update this, it does not need to float.
+  # the most recent release rhel-9.2 at present. since we yum update this, it does not need to float.
   # it is important that we not build from unreleased builds and publish them.
-  # rhel-els-container-9.0.0-1228
-  image: registry-proxy.engineering.redhat.com/rh-osbs/rhel-els@sha256:47aa1b5572e41f79537afce517c7edbad6ae592f7ab819b1ce9e7d22293ef0ff
+  # ubi9-container-9.2-489
```

Comment:
[rhel-els-container-9.2-483](https://brewweb.engineering.redhat.com/brew/buildinfo?buildID=2489460) shipped today, probably best to base it on that from the start.